src/components/victory-area/victory-area.js

```diff
@@ -484,7 +521,8 @@ export default class VictoryArea extends React.Component {
 
     if (animate) {
       const whitelist = [
-        "data", "domain", "height", "padding", "style", "width", "clipWidth", "clipHeight"
+        "data", "domain", "height", "padding", "style", "width",
+        "clipWidth", "clipHeight", "translateX"
```

Comment:
actually, I think `clipWidth`, `clipHeight`, and `translateX` don't belong here  because they are not props for `VictoryArea`, so they aren't passed into `VictoryTransition` anyway.